libraries/aws_backend.rb

```diff
@@ -480,6 +480,10 @@ def method_missing(method_name, *args, &block)
   def respond_to_missing?(*several_variants)
     super
   end
+
+  def to_s
+    "Missing property extension. Followings can be tested via dot notation: #{item.keys.map(&:to_s)}"
```

Comment:
I'm unsure of the wording on this. Perhaps something more along the lines of:
```
"Missing property extension. The following properties are currently available: #{item.keys.map(&:to_s)}"
```

Reply:
Agreed, I'd suggest:
```
     "Property is missing!  The following are available: #{item.keys.map(&:to_s)}"
```
just because it's less interesting for the consumer about the property extension.